memory/utils/gaussdb_conn.py

```
@@ -0,0 +1,1603 @@
+        for document in documents:
+            doc_id = str(document.get("id") or "")
+            try:
+                # One batch must use one embedding dimension because a MERGE can
+                # bind only one q_{dim}_vec column. Reject mixed dimensions before
+                # executing SQL and return the failed IDs.
+                row, row_dim = self._message_to_row(document, memory_id)
+                dim = row_dim if dim is None else dim
+                if row_dim != dim:
+                    raise ValueError(f"inconsistent content_embed dimension: expected {dim}, got {row_dim}")
+                rows.append(row)
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)